tests/modules/internet_test.cpp

```diff
@@ -803,3 +804,9 @@ TEST_F(InternetTest, shouldGenerateAnonymousUsernameWithMaxLength)
 
     ASSERT_EQ(generatedUsername.length(), 20);
 }
+
```

Comment:
please add more tests, each public function should be tested, in jwt function try to test different parameters also

Reply:
sure so the helper functions coud stay puplic ?

Reply:
I think they could be moved to anonymous namespace in cpp file as they are not public API